code/modules/reagents/reagents/core.dm

```diff
@@ -96,11 +96,41 @@
 	glass_name = "water"
 	glass_desc = "The father of all refreshments."
 	nerve_system_accumulations = 0
+	var/fire_supperion_affect = 1 //19000 times this.
 	reagent_type = "Water"
 
+/datum/reagent/water/extinguisher
+	name = "Extinguisher"
+	id = "abwater"
+	description = "A mix of water, with chemicals to reduce heat, oil and suppress fire with heavy particulates."
+	taste_description = "watered down chemicals"
+	glass_icon_state = "glass_clear"
+	glass_name = "water"
+	glass_desc = "The father of all refreshments, this one has floating particulates in it..."
+	nerve_system_accumulations = 50 //Chemical soup
```

Comment:
If it has NSA make it at least toxic, you wouldn't want to drink extinguisher mix do you....?

Reply:
Fire mix iirc is FDA rated as a non-toxic.